appcues/src/main/java/com/appcues/util/StringExt.kt

```diff
@@ -6,9 +6,11 @@ internal fun String.toSlug() = lowercase(Locale.getDefault())
     .replace("\n", " ")
     .replace("[^a-z\\d\\s]".toRegex(), " ")
     .split(" ")
+    .filter { it.isNotEmpty() }
     .joinToString("-")
     .replace("-+".toRegex(), "-")
```

Comment:
I think the filter makes this regex redundant since there should never be `--` any more? Probably fine to leave though.

Reply:
yeah, I agree it should really be redundant now. I'll leave it just to be safe I suppose.